### PR TITLE
Experiment data etl + api endpoint

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

File: missioncontrol/base/models.py [hunk 1]
```diff
@@ -23,14 +23,42 @@ class Meta:
         db_table = 'channel'
 
 
+class Experiment(models.Model):
+    '''
+    Represents an experiment
+    '''
+    name = models.CharField(max_length=100, unique=True)
```
Comment (robhudson) [inline missioncontrol/base/models.py]: I believe experiment names are `max_length=255` in normandy.

**Comment by wlach** [comment]
Example query:

http://localhost:8000/api/experiment/?measure=main_crashes&interval=200000&experiment=pref-flip-activity-stream-58-beta-pocket-personalization-bug-1425490

Example response:

https://gist.github.com/wlach/751f81dacf278e1b8ac4f8eee6e4b49c

File: missioncontrol/base/models.py (hunk 1 above)
Comment (wlach) [reply missioncontrol/base/models.py]: Good catch! Will fix

**Comment by wlach** [comment]
@robhudson ok, I think this is ready for actual review now. hopefully the above is helpful for understanding what this does

File: missioncontrol/settings.py [hunk 2]
```diff
@@ -311,6 +323,8 @@
 FIREFOX_VERSION_URL = 'https://product-details.mozilla.org/1.0/firefox_versions.json'
 FIREFOX_VERSION_CACHE_TIMEOUT = 300
 
+FIREFOX_EXPERIMENTS_URL = 'https://normandy.cdn.mozilla.net/api/v1/recipe/signed/?enabled=true'
```
Comment (robhudson) [inline missioncontrol/settings.py]: At least in experiments viewer the only ones we're interested in have an extra filter:
`https://normandy.cdn.mozilla.net/api/v1/recipe/signed/?enabled=true&latest_revision__action=3`

File: missioncontrol/etl/tasks.py [hunk 3]
```diff
@@ -18,3 +22,35 @@ def update_measures():
             for measure_name in platform['measures']:
                 update_measure.apply_async(
                     args=[platform_name, channel_name, measure_name])
+
+
```
Comment (robhudson) [inline missioncontrol/etl/tasks.py]: Just a nit: The names of these 2 tasks sound very similar. Something like `update_experiment_data` seems to fit and describe what the task is doing.

**Comment by robhudson** [comment]
Is it possible to document the API, similar to what we do on EV?
https://github.com/mozilla/experiments-viewer/blob/master/docs/api.md

**Comment by wlach** [comment]
I will add some docs with this PR. I like the approach of a simple markdown document, at least for the initial stages.

**Pull request merged**